base/libgit2/walker.jl

```diff
@@ -83,6 +82,27 @@ end
 
 repository(w::GitRevWalker) = w.owner
 
+"""
+    LibGit2.map(f::Function, walker::GitRevWalker; oid::GitHash=GitHash(), by::Cint = Consts.SORT_NONE, rev::Bool=false)
```

Comment:
Inconsistent spacing around `=`. In this case I'd remove the spaces between `Cint` and `Consts`.